tailwindui-storybook.js

```diff
@@ -52,12 +47,15 @@ async function run() {
     components.forEach((component) => {
       const componentName = pascalCase(component.title);
 
+      // Add `Component_` prefix for import name if component name starts with number
+      const importName = componentName.match(/^\d/) ? `Component_${componentName}` : componentName;
```

Comment:
JS variable can't start with number and so is JSX tag. So I added custom prefix `Component_` when the imported component starts with number.

Before / after:
![image](https://user-images.githubusercontent.com/7823011/199605357-e83d19dc-d91d-4033-9b1b-e661dbff16cf.png)